src/adapters/MeetingsJSONAdapter.test.js

```diff
@@ -166,6 +170,11 @@ describe('Meetings JSON Adapter', () => {
       expect(testMeeting.remoteVideo).toBeInstanceOf(MediaStream);
     });
 
+    test('sets meeting state to JOINED', async () => {
```

Comment:
let's have an expect() before the action, so that we check that the state really has changed. Otherwise, maybe the state was 'JOINED' from the beginning.